docs/extension/create/README.md

```diff
@@ -8,35 +8,38 @@ Creates an extension
 Usage: contentful extension create [options]
 
 Options:
-  --id           ID                                                [string]
-  --name         Name                                   [string] [required]
-  --space-id     Space ID                               [string] [required]
-  --field-types  Field types                             [array] [required]
-  --descriptor   Descriptor                                        [string]
-  --src          Src                                               [string]
-  --srcdoc       Srcdoc                                            [string]
-  --sidebar      Sidebar                                          [boolean]
+  --id                       ID                                    [string]
+  --name                     Name                       [string] [required]
+  --space-id                 Space ID                   [string] [required]
+  --field-types              Field types                 [array] [required]
+  --descriptor               Path of descriptor file               [string]
+  --src                      Src                                   [string]
+  --srcdoc                   Path of srcdoc file                   [string]
+  --sidebar                  Renders in sidebar                   [boolean]
+  --installation-parameters  Installation parameter values         [string]
```

Comment:
key-values rather than values?

Reply:
There's an exmaple below and we say "key-value pairs" in the output. It's already the longest line.